src/EnergyPlus/DataGlobalConstants.hh

```diff
@@ -181,88 +178,11 @@ namespace DataGlobalConstants {
     Real64 constexpr StefanBoltzmann = 5.6697E-8;           // Stefan-Boltzmann constant in W/(m2*K4)
     Real64 constexpr UniversalGasConst = 8314.462175;       // Universal Gas Constant (J/mol*K)
     Real64 constexpr convertJtoGJ = 1.0E-9;                 // Conversion factor for J to GJ
-    int constexpr MaxSpeedLevels = 10;                      // Maximum number of speed that supports
-    int constexpr ScheduleAlwaysOn = -1;                    // Value when passed to schedule routines gives back 1.0 (on)
-    int constexpr MaxCTFTerms = 19; // Maximum number of CTF terms allowed to still allow stability //Note Duplicate of DataHeatBalance::MaxCTFTerms
 
     ResourceType AssignResourceTypeNum(std::string const &ResourceTypeChar);
     std::string GetResourceTypeChar(ResourceType ResourceTypeNum);
 
-} // namespace DataGlobalConstants
-
-struct DataGlobalConstantsData : BaseGlobalStruct
```

Comment:
These data structures are not used anymore, therefore this entire `struct` is not needed.